packages/react/src/prop-types/childrenOf.js

```diff
@@ -21,14 +22,20 @@ const childrenOf = expectedChildTypes => {
   const expectedDisplayNames = expectedChildTypes
     .map(child => getDisplayName(child.type || child))
     .join(', ');
+  const expectedChildElementTypes = expectedChildTypes.map(
+    child => React.createElement(child).type
```

Comment:
We often go down this road but get stumped when props are marked as required for a child and this call will trigger warnings as a result 😞 Does this PR offer a way around that, or is this not a problem anymore?

Reply:
nope, still a problem as of now. See my previous comment. and @asudoh 's reply.

Reply:
Thanks @finken2!